lms/static/js/learner_dashboard/views/course_card_actions.js

```diff
@@ -0,0 +1,41 @@
+;(function (define) {
+    'use strict';
+
+    define(['backbone',
+            'jquery',
+            'underscore',
+            'gettext',
+            'edx-ui-toolkit/js/utils/html-utils',
+            'text!../../../templates/learner_dashboard/course_card_actions.underscore'
+           ],
+         function(
+             Backbone,
+             $,
+             _,
+             gettext,
+             HtmlUtils,
+             pageTpl
+         ) {
+            return Backbone.View.extend({
+                tpl: HtmlUtils.template(pageTpl),
+
+                events: {
+                    'click .enroll-button': 'handleEnroll'
+                },
+
+                initialize: function() {
+                    this.render();
+                },
+
+                render: function() {
+                    var filledTemplate = this.tpl(this.model.toJSON());
+                    HtmlUtils.setHtml(this.$el, filledTemplate);
+                },
+
+                handleEnroll: function(){
+                    //Enrollment click event handled here
+                }
+            });
+        }
+    );
+}).call(this, define || RequireJS.define);
```

Comment:
In my design, I want this view to render both the `enroll now` button as well as the `View Course` button after learner enrolled. Therefore I made the view to have such a name. Maybe the design I have can be improved. Since this view is really out of the scope for the ticket for now, can we come back to this area in our following tickets and discuss again?